script.js

```diff
@@ -0,0 +1,105 @@
+// DOM Elements
```

Comment:
You don't have to explain such declarations as a comment. The meaning here is quite obvious and understandable.

Reply:
DONE!